app/forms/household_relationship_form.py

```diff
@@ -17,16 +18,17 @@ def build_relationship_choices(answer_store, group_instance):
     :param group_instance: The instance of the group being iterated over
     :return:
     """
-    household_answers = answer_store.filter(block_id='household-composition')
+    household_first_name_answers = answer_store.filter(block_id='household-composition', answer_id='first-name', limit=settings.EQ_MAX_NUM_REPEATS)
```

Comment:
Well, that will just strip it down to the empty string '' and is only for mandatory answers...so won't affect middle or last names. Middle and last names are stripped by the optional validator, but don't have to be populated and store blank entries. You can see some existing tests which haven't been altered in my branch for such behaviour here: https://github.com/ONSdigital/eq-survey-runner/blob/master/tests/app/forms/test_household_composition.py#L62